components/ai_chat/core/browser/engine/engine_consumer_claude.cc

```diff
@@ -98,7 +116,8 @@ std::string BuildClaudePrompt(
   std::string date_and_time_string =
       base::UTF16ToUTF8(TimeFormatFriendlyDateAndTime(base::Time::Now()));
   std::string prompt = base::StrCat(
-      {kHumanPromptSequence, prompt_segment_article,
+      {kHumanPromptSequence, prompt_segment_selected_text,
+       prompt_segment_article,
```

Comment:
It seems like we're connecting the latest question to the selected text in the wrong part of the prompt - this is the system part - the introduction before the conversation history and well before the latest/current human question. The way we're doing it in `GetConversationHistoryString` seems most appropriate - why not do it like that also for the latest question?
Otherwise it seems like the whole conversation is being tied to that excerpt and that might not lead to the AI restricting the answer to that excerpt especially when there is already a conversation. Or am I missing something?